traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -23,13 +23,22 @@ var TableServersController = function(tableName, servers, filter, $scope, $state
 	// browserify can't handle classes...
 	function SSHCellRenderer() {}
 	SSHCellRenderer.prototype.init = function(params) {
-		this.eGui = document.createElement("A");
-		this.eGui.href = "ssh://" + userModel.user.username + "@" + params.value;
-		this.eGui.setAttribute("target", "_blank");
+		this.eGui = document.createElement("div");
 		this.eGui.textContent = params.value;
+		this.data.cellHrefValue = "ssh://" + userModel.user.username + "@" + params.value;
```

Comment:
Getting `this.data` is undefined on page load
```
Uncaught TypeError: this.data is undefined
    init https://localhost/resources/assets/js/app.js?built=1606856003402:24254
    initComponent https://localhost/resources/assets/js/ag-grid-community/dist/ag-grid-community.min.js:8
    createAndInitUserComponent https://localhost/resources/assets/js/ag-grid-community/dist/ag-grid-community.min.js:8
    newCellRenderer https://localhost/resources/assets/js/ag-grid-community/dist/ag-grid-community.min.js:8
    createCellRendererFunc https://localhost/resources/assets/js/ag-grid-community/dist/ag-grid-community.min.js:8
    executeFrame https://localhost/resources/assets/js/ag-grid-community/dist/ag-grid-community.min.js:8
    requestFrame https://localhost/resources/assets/js/ag-grid-community/dist/ag-grid-community.min.js:8
app.js:24254:3```

Reply:
This literally crossed my mind Saturday afternoon :upside_down_face: 

I need to figure out where it's approps to stash this, the context presently isn't correct of course